app/src/main/kotlin/com/intive/picover/parties/model/PartyRemote.kt

```diff
@@ -8,3 +8,9 @@ data class PartyRemote(
 	val title: String,
 	val description: String,
 )
+
+fun Party.toRemote() = PartyRemote(
+	id = id,
+	title = title,
+	description = description,
+)
```

Comment:
1. Is there a particular reason to define it as an extension?
2. Think about dependency between these two classes. Is it a good place for such mapping? Maybe better would be to define it in `Party` file?

Reply:
@marcinWisnia What do you think, I think we have discussed it preiviously.

Reply:
@kmaslowiec  @MarekMacko sorry, my mistake - i must have read something wrong - it should be in Party class as it is a "part" of this class functionality, that's my real feeling  :D.  

According to having it as an extension, i don't think it is really needed, but if someone has another opinion, please share :D

Reply:
Take into consideration potential coupling between these two classes. We have only one module but remote classes shouldn't know anything about UI. In the same way repository knowns nothing about activity or fragment, or at least shouldn't know :)

Reply:
@MarekMacko I agree. I've moved the Party ext. to Party class and same with the RemoteParty.

Reply:
We've started discussion about placing extension `Party.toRemote()` in `PartyRemote.kt` file. Conclusion is to keep such extension in `Party.kt` file. 
Extension `PartyRemote.toUI` should stay as it was.